File: arrays/twoSum/twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        num_dict = {}
        for i in range(len(nums)):
            num = nums[i]
            find = target - num
            if find in num_dict:
                return [i,num_dict[find]]
            num_dict[num] = i

File: arrays/twoSum/twoSum.ts
function twoSum(nums: number[], target: number): number[] {
    const dict: Record<number, number> = {};
    for (let i = 0; i < nums.length; i++) {
        const find  = target - nums[i];
        if (find in dict) {
            return [dict[find], i];
        }
        dict[nums[i]] = i;
    }
    return [];
};